modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/css/main.scss

```diff
@@ -48,31 +50,36 @@
 }
 
 // Sidebar
-.sidebar-wrapper {
-	height: 100%;
-	width: 0;
 
-	&.open {
-		width: 320px;
-	}
-}
+$sidebarWidth: 320px;
 
-.sidebar-wrapper,
-.dashboard-content {
-	transition: 0.5s;
+.has-control-menu .sidebar {
+	top: 56px;
 }
 
 .sidebar {
-	.sidebar-header {
-		padding-bottom: 5px;
-		padding-top: 10px;
+	bottom: 0;
+	left: 100%;
+	position: fixed;
+	width: $sidebarWidth;
+	z-index: $zindex-dropdown - 1;
```

Comment:
Pray that, er, nobody complains about the mixed variable style here (camel and kebab).

Reply:
🤷🤷 🤷 🤷 🤷 🤷 



![image](https://user-images.githubusercontent.com/5803434/88531171-b72e5380-d002-11ea-8cf6-3518f5c9042f.png)

Reply:
I think @wincent meant `$zindex-dropdown`?

Reply:
Looks like all variables in Clay are hyphen-based... was looking but we don't have any official guidelines for this. @wincent, might need to proactivley lint this? 😂

Reply:
> I think @wincent meant `$zindex-dropdown`? Which is coming from Clay? 🤔

Well, I just meant that one of them is not like the other; I didn't search to see which one was actually more prevalent.

I was just flagging it because it might stand out to Brian, seeing as the variables are right next to each other. Don't want to get...

![denied](https://user-images.githubusercontent.com/7074/88541690-7c81e680-d015-11ea-9e9b-a593d37b71e4.gif)

> @wincent, might need to proactivley lint this?

Yes, although if we have a zillion of each type it might require quite a bit of manual clean-up first... could be tricky, but I created an issue for it anyway: https://github.com/liferay/liferay-npm-tools/issues/476

Reply:
As bad as it sounds... this being used in only 2 places, maybe it'd be best to look the other way and hardcode those beautiful magic numbers 😂

Reply:
Camel Case variable names are from the Bootstrap 2 days. Bootstrap 3 changed their pattern from camel case to kebab case.